config/values-schema.yml

```diff
@@ -0,0 +1,18 @@
+#@data/values-schema
```

Comment:
We should align our values as closely as we can with TCE:

https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/kapp-controller/0.44.1/bundle/config/values.yaml

This can be done in this PR or a follow-up. But the downstream consumers of TCE will expect the values of their kc package to align, or else they will have to add an overlay. For secretgen-controller we chose to align as much as we felt was valuable to consumers to expose.

Reply:
So you mean we should have all these new values as part of our values-schema.yml or the value fields we already have should have exact same value as per TCE values.yaml?

Reply:
> So you mean we should have all these new values as part of our values-schema.yml 

Right, not that we *should* have them all, so much as we have a direct consumer that needs these things configurable via values. So we should consider them in our official values schema. Otherwise downstream consumers will have to add an overlay to expose these values

Reply:
Done added the values aligning with TCE